common/spec/dependabot/config/ignore_condition_spec.rb

```diff
@@ -116,6 +148,15 @@ def expect_ignored(versions)
         it "returns the expected range" do
           expect(ignored_versions).to eq([">= 2.a, < 3"])
         end
+
+        context "with a single major dependency version" do
+          let(:dependency_version) { "1" }
+
+          it "ignores patch updates" do
+            expect_ignored(["2.0.0", "2.0.0.a"])
+            expect_allowed([dependency_version, "1.2.2", "1.2.0.a", "3.0.0"])
```

Comment:
This looks like existing behavior but do we actually want to allow 3? I'd expect if I ignore major versions I won't get v2, v3, v4, etc.

Reply:
Good spot! I just spotted this when testing in the wild: https://github.com/github/dependabot-updates/issues/1350#issuecomment-829374042

I think you're right, we should ignore all major versions. We have the current behaviour when ignore from PRs as you're talking about a specific major version.